scripts/calibrateArmDac.sh

```diff
@@ -34,71 +59,121 @@ then
     return
 fi
 
-echo -e "ChamberName\tscandate\tCFG_THR_ARM_DAC" 2>&1 | tee listOfScanDates_calibrateArmDac_${DETECTOR}.txt
+if [ -z ${RESUMEDIR} ];
+then
+    scandate=$(date +%Y.%m.%d.%H.%M)
+    OUTDIR=${DATA_PATH}/${DETECTOR}/armDacCal/${scandate}
+    mkdir -p $OUTDIR
```

Comment:
I would edit the permissions on this directory to have group read and write the moment you make it.  While at the end of the script you apply this recursively if an issue occurs that prevents the script from executing to completion (e.g. `KeyboardInterrupt`) the permissions will not be set correctly.